### ISSUE TITLE
Remove the inscription “Better Saved Hotbars Update for different version available...”

### ISSUE THREAD

**Issue by xleb-normal** [LukynkaCZE/better-saved-hotbars#28]
Remove the inscription “Better Saved Hotbars Update for different version available...” if the new version of the mod does not fit the client version or add the ability to remove this inscription through configs

**Comment by Keaton42** [comment]
I fully agree with this, or at least an option to disable it. I play on the latest version of this mod for 1.21.1 and literally have to change versions to update it

**Comment by KostyaPictures** [comment]
<img width="1386" height="747" alt="Image" src="https://github.com/user-attachments/assets/9855cd1b-0359-4546-8252-57a370d75bb4" />
Agree, also it overlaps on other mod texts
